add link to forum in README (#17)
Unity-Technologies/Unity-Simulation-Docs: README.md

# Unity Simulation Documentation

> Welcome and thanks for checking out Unity Simulation. Unity Simulation allows you to run different simulations using Unity. Unity Simulation is in active development. We hope you enjoy using the service, if you have any questions please reach us at [simulation-help](mailto:simulation-help@unity3d.com) or open a thread in [our forum](https://forum.unity.com/forums/unity-simulation.407/).

Reviewer: ```suggestion
Welcome and thanks for checking out Unity Simulation. Unity Simulation allows you to run configurable simulations using Unity. Unity Simulation is in active development. We hope you enjoy using the service, if you have any questions please open a thread in [the forum](https://forum.unity.com/forums/unity-simulation.407/) or contact support at [simulation-help](mailto:simulation-help@unity3d.com).
```
